guides/gatherers.md

```diff
@@ -1078,6 +1079,41 @@ Example output (in Rhai):
 }
 ```
 
+<span id="sapservicesv1"></span>
+
+### sapservices@v1
```

Comment:
Alphabetical order please....
This should come before `saptune`

Reply:
done